### DISCUSSION TITLE
Webserver: Further Sanitize values passed to origin param

### DISCUSSION THREAD

**Discussion by mingkeming** [apache/airflow#13220] [Q&A] [answered]
in "https://nvd.nist.gov/vuln/detail/CVE-2020-17515" I found this vulnerability has not solve completely. is this the same to  issure "Webserver: Further Sanitize values passed to origin param"? and has this vulnerability solved in 1.10.14?

**Comment by kaxil** [comment, marked as answer]
Yes this has been completely fixed in 1.10.14

**Reply by kaxil** [reply]
https://lists.apache.org/x/thread.html/r2892ef594dbbf54d0939b808626f52f7c2d1584f8aa1d81570847d2a@%3Cannounce.apache.org%3E
